src/components/v2/common/ReactSelect.utils.tsx

```diff
@@ -32,14 +33,28 @@ export const styles = {
 }
 
 export function Option(props) {
-    const { selectOption, data } = props;
+    const { selectOption, data, showTippy } = props
     const style = { height: '16px', width: '16px', flex: '0 0 16px' }
-    const onClick = (e) => selectOption(data);
-    return <div className="flex left pl-12" style={{ background: props.isFocused ? 'var(--N100)' : 'transparent' }}>
-        {props.isSelected ? <Check onClick={onClick} className="mr-8 icon-dim-16" style={style} />
-            : <span onClick={onClick} className="mr-8" style={style} />}
-        <components.Option {...props} />
-    </div>
+    const onClick = (e) => selectOption(data)
+    
+    const getOption = () => {
+        return <div className="flex left pl-12" style={{ background: props.isFocused ? 'var(--N100)' : 'transparent' }}>
+                {props.isSelected ? (
+                    <Check onClick={onClick} className="mr-8 icon-dim-16" style={style} />
+                ) : (
+                    <span onClick={onClick} className="mr-8" style={style} />
+                )}
+                <components.Option {...props} />
+            </div>
+    }
+
+    return showTippy ? (
+        <Tippy className="default-white ml-10" arrow={false} placement="right" content={data.label}>
+            {getOption()}
+        </Tippy>
+    ) : (
+        <>{getOption()}</>
```

Comment:
We don't have to wrap it within fragments here. We can simply say `{getOption()}`.

Reply:
Its showing error

Reply:
It'll work :)

Try -> `return showTippy ? (<Tippy></Tippy>) : getOption()`